src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -107,6 +107,12 @@ public string ClientVersion
 
         public bool RetryOnServerErrors { get; set; } = true;
 
+        public bool UseManagedIdentity { get; internal set; }
+
+        public string MIUserAssignedClientId {  get; internal set; }
+
+        public string MIUserAssignedResourceId { get; internal set; }

```

Comment:
I would spell these out - `ManagedUserIdentityClientId`, `ManagedUserIdentityResourceId`. It's longer but I think clearer, since `MI` may not be too obvious what it means.